python/python/glide/async_commands/server_modules/json.py

```diff
@@ -560,6 +560,54 @@ async def nummultby(
     return cast(Optional[bytes], await client.custom_command(args))
 
 
+async def objlen(
+    client: TGlideClient,
+    key: TEncodable,
+    path: Optional[TEncodable] = None,
+) -> Optional[TJsonResponse[int]]:
+    """
+    Retrieves the number of key-value pairs in the object values at the specified `path` within the JSON document stored at `key`.
+
+    Args:
+        client (TGlideClient): The client to execute the command.
+        key (TEncodable): The key of the JSON document.
+        path (Optional[TEncodable]): Represents the path within the JSON document where the key names will be retrieved.
+            Defaults to None.If not provided, the root of the document is used, equivalent to setting the path to ".".
```

Comment:
```suggestion
        path (Optional[TEncodable]): The path within the JSON document. Defaults to None.
```
what youre asying is true to almost all the commands, we will think of something to write, meanwhile lets stick with this